README.md

```diff
@@ -1 +1,119 @@
-# catalog-of-my-things
\ No newline at end of file
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [Catalog of my things] <a name="about-project"></a>
+
+**[Catalog of my things]** is a console app project, that will help you to keep a record of different types of things you own: books, music albums, and games. The data will be stored in JSON files.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+<summary>Catalog of my things app built with Ruby.</summary>
+  
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Add books**
+- **Add games**
+- **Add albums**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Video presentation](https://drive.google.com/file/d/1fpKi5678kdljgkF2E-YdDqBEuovNRckl/view?usp=sharing)
```

Comment:
Hello guys, you have built a very beautiful application, well done 👏🏾 . 

- [ ] But based on the project requirement, the video length should not exceed 5mins. Note below the things to consider when re-doing this video:
- The maximum length of the video should be 5 minutes.
- The video contains a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
- All the members in the video should appear in the video and talk about the project.
- The video contains a highlight about a piece of the code the student finds interesting or is proud of making. 💯 

Kindly redo your video according to those guidelines.

Reply:
Thank you @bobb-Rob for your review.